.github/workflows/review-application.yml

```diff
@@ -0,0 +1,29 @@
+name: Review application
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  review-application:
+    runs-on: ubuntu-latest
+    if: contains(github.event.issue.labels.*.name, 'application')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch processor
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          file: "processor"
+          target: "./processor"
+
+      - name: Automated application review
+        run: |
+          chmod +x ./processor
```

Comment:
GH retains permissions in artifact uploading, so if it was uploaded with execute, it still should be here. `dsaltares/fetch-gh-release-asset` might not respect the permissions though.